qa/pom.xml

```diff
@@ -248,6 +260,29 @@
             <artifactId>slf4j-api</artifactId>
             <scope>provided</scope>
         </dependency>
+        
+        <!-- needed by Elasticsearch -->
+        <dependency>
+            <groupId>org.elasticsearch</groupId>
+            <artifactId>elasticsearch</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.elasticsearch.client</groupId>
+            <artifactId>transport</artifactId>
+            <version>${elasticsearch-client-transport.version}</version>
+        </dependency>
+        <dependency>
+	    	<groupId>org.apache.logging.log4j</groupId>
+	    	<artifactId>log4j-api</artifactId>
+	    	<version>${log4j-to-slf4j.version}</version>
+	    	<scope>test</scope>
+		</dependency>
+		<dependency>
+	    	<groupId>org.apache.logging.log4j</groupId>
+	    	<artifactId>log4j-core</artifactId>
+	    	<version>${log4j-to-slf4j.version}</version>
```

Comment:
Sorry, I reviewed the dependencies and now it's clear you appointment. May I did something wrong during the rebases/merge I did beacuse the log4j core is present in few pom.
Since the log4j core should be replaced by
<dependency>
    <groupId>org.apache.logging.log4j</groupId>
    <artifactId>log4j-to-slf4j</artifactId>
</dependency>
there is no need to have it in the dependencies.
Right?